src/Makefile.am

```diff
@@ -32,7 +32,7 @@ BUILT_SOURCES = \
 	eos-updater-generated.c \
 	$(NULL)
 
-libeos_updater_dbus_la_CPPFLAGS = $(common_cppflags)
+libeos_updater_dbus_la_CPPFLAGS = $(common_cppflags) -DG_LOG_DOMAIN=\""EosUpdaterDBus"\"
```

Comment:
Nitpick: `libeos-updater-dbus`.

Similarly for the other domains in the patch (I won’t leave more comments about this).